app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnitsMap/index.tsx

```diff
@@ -354,6 +368,24 @@ function NationalSocietyLocalUnitsMap(props: Props) {
                     </MapPopup>
                 )}
             </BaseMap>
+            <Container
+                className={styles.mapDetail}
+                childrenContainerClassName={styles.addressDetail}
+            >
+                <div className={styles.location}>
+                    <div>{countryResponse?.address_1}</div>
+                    <div>{countryResponse?.address_2}</div>
+                    <div>{countryResponse?.city_code}</div>
+                </div>
+                <Link
+                    href={`mailto:${countryResponse?.email ?? '-'}`}
+                    variant="tertiary"
+                    withUnderline
+                    external
+                >
+                    {countryResponse?.email}
+                </Link>
```

Comment:
It's better to check if we have the email before assigning it to the link
```suggestion
                {isDefined(countryResponse) && isDefined(countryResponse.email) && (
                    <Link
                        href={`mailto:${countryResponse.email}`}
                        variant="tertiary"
                        withUnderline
                        external
                    >
                        {countryResponse?.email}
                    </Link>
                )}
```